<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="ATMECSINDT-049" name="Test" tests="8" failures="2" timestamp="22 Aug 2014 13:13:03 GMT" time="0.353" errors="0">
  <testcase name="test1" time="0.035" classname="com.atmecs.com.atmecs.ReportCreator.TestClass"/>
  <testcase name="test2" time="0.0" classname="com.atmecs.com.atmecs.ReportCreator.TestClass"/>
  <testcase name="test3" time="0.009" classname="com.atmecs.com.atmecs.ReportCreator.TestClass">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1]
	at com.atmecs.com.atmecs.ReportCreator.TestClass.test3(TestClass.java:27)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase name="test4" time="0.001" classname="com.atmecs.com.atmecs.ReportCreator.TestClass"/>
  <testcase name="test5" time="0.001" classname="com.atmecs.com.atmecs.ReportCreator.TestClass"/>
  <testcase name="test6" time="0.001" classname="com.atmecs.com.atmecs.ReportCreator.TestClass"/>
  <testcase name="test7" time="0.001" classname="com.atmecs.com.atmecs.ReportCreator.TestClass">
    <failure type="java.lang.AssertionError" message="expected [0] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [0] but found [1]
	at com.atmecs.com.atmecs.ReportCreator.TestClass.test7(TestClass.java:51)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- test7 -->
  <testcase name="test8" time="0.001" classname="com.atmecs.com.atmecs.ReportCreator.TestClass"/>
</testsuite> <!-- Test -->
